kitsune/wiki/config.py

```diff
@@ -28,6 +28,20 @@
         u'como-fazer-o-firefox-parar-de-se-conectar-automati',
     ]
 }
+FALLBACK_LOCALES = {
+    'bn-IN': ['bn-BD'],  # Bug 800880
```

Comment:
Please also add this line below:
'bn-BD': ['bn-IN'],  # Bug 800880
as per discussion with Swarnava